back/src/controllers/Collab.controller.ts

```diff
@@ -0,0 +1,25 @@
+import { CraApplication } from "../domain/application/craApplication";
+import { Collab } from "../domain/model/Collab";
+import { Body, Controller, Get, Post } from "@nestjs/common";
+
+@Controller('collab')
+export class CollabController{
+
+    constructor(private readonly craApplication:CraApplication) {}
+
+
+    @Get('all')
+    async getCollabss(): Promise<Collab[]> {
+        console.log("getting collabs back");
+        return await this.craApplication.getAllCollabs();
+      
+    }
+
+    @Post('ids')
+    async getCollabsByIds(@Body() ids:string[]): Promise<Collab[]> {
+        console.log("getting collabs back by ids");
```

Comment:
Nestjs has a logger service. Please use it and please make good use of the logger levels "debug", "info" etc

Reply:
Also when logging it is a good practice to add the params inside the log for easier debugging unless these params are sensitive